lib/pages/portal/view/website_view.dart

```diff
@@ -323,33 +397,44 @@ class _WebsiteViewState extends State<WebsiteView> {
 }
 
 class WebsiteIcon extends StatelessWidget {
-  const WebsiteIcon({this.website, this.canEdit, this.size, this.onTap});
+  const WebsiteIcon(
+      {this.website, this.canEdit, this.size, this.image, this.onTap});
 
   final Website website;
   final bool canEdit;
   final double size;
+  final ImageProvider image;
   final Function onTap;
 
   @override
   Widget build(BuildContext context) {
-    return FutureBuilder(
-      future: StorageProvider.findImageUrl('websites/${website.id}/icon.png'),
-      //Firebase Storage path
-      builder: (context, snapshot) {
-        ImageProvider image;
-        image = const AssetImage('assets/icons/globe.png');
-        if (snapshot.hasData) {
-          image = NetworkImage(snapshot.data);
-        }
+    if (image == null) {
+      return FutureBuilder(
+          future:
+              StorageProvider.findImageUrl('websites/${website.id}/icon.png'),
+          //Firebase Storage path
```

Comment:
I know you didn't write this, but add a space between // and the comment as that's the proper style (I'm surprised the linter doesn't pick this up - note to self: try to add an automated check for this).

Reply:
Done on my part - https://github.com/student-hub/acs-upb-mobile/pull/236